### PR TITLE
Update package.json and LICENSE

### PR DESCRIPTION
**Pull request by geoffturk**
[PR Process](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-process) - [PR Review Checklist](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-review-checklist)

## Description of Changes

Update `package.json` and `LICENSE` files with MIT license and other changes.

### Warning

Merging to `master` automatically triggers a deploy in production.

### CODE REVIEW

**Review by warrenv — APPROVED** [review]
* [n/a] Tested changes manually
* [n/a] Checked accidental architectural/style changes
* [x] Reviewed entire diff
* [n/a] Unit tests
* [x] Documentation
* [x] Filenames and locations

**Review by kennylavender — APPROVED** [review]
* [n/a] Tested changes manually
* [n/a] Checked accidental architectural/style changes
* [x] Reviewed entire diff
* [n/a] Unit tests
* [n/a] Documentation
* [n/a] Filenames and locations

File: package.json [hunk 1]
```diff
@@ -21,10 +21,15 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/poetapp/explorer-web.git"
+  },
   "license": "MIT",
-  "greenkeeper": {
+  "bugs": {
+    "url": "https://github.com/poetapp/explorer-web/issues"
+  },
+   "greenkeeper": {
```
Comment (kennylavender) [inline package.json:32]: seems to be an extra space here before greenkeeper.

Comment (geoffturk) [reply package.json:32]: Ah, OK.

Comment (geoffturk) [reply package.json:32]: Was already closed so I created a new PR.

**Pull request merged**